mainwindow.cpp

```diff
@@ -638,6 +663,14 @@ QStringList MainWindow::saveAttackParameters()
     if (m_ui->checkBox_EnvironmentVar->isChecked()) {
         m_settings.setValue("environmentVariables", m_ui->lineEdit_EnvironmentVar->text());
     }
+    // Save unselected rows
+    QList<QVariant> unselectedRows;
```

Comment:
No, since selection of hash was related a lot with filters, I merged and continued work on #64